google/cloud/storage/internal/grpc/metrics_exporter_impl.cc

```diff
@@ -44,6 +56,29 @@ auto MakeReaderOptions(Options const& options) {
   return reader_options;
 }
 
+class ExporterRegistry {
+ public:
+  ExporterRegistry() = default;
+
+  static ExporterRegistry& Singleton() {
```

Comment:
to quote jgm: "Singleton..." \*shudders\*

Reply:
This should go away in gRPC 1.65 or 1.66. I expect we will be able to configure these per-channel.